### PR TITLE
Add missing urls in collections module

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @steveklabnik

### CODE REVIEW

File: src/libstd/collections/mod.rs [hunk 1]
```diff
@@ -263,42 +263,42 @@
...
-//! the only valid operation is to `insert` a value into the entry. When this is
+//! the only valid operation is to `insert()` a value into the entry. When this is
 //! done, the vacant entry is consumed and converted into a mutable reference to
 //! the value that was inserted. This allows for further manipulation of the
 //! value beyond the lifetime of the search itself. This is useful if complex
 //! logic needs to be performed on the value regardless of whether the value was
 //! just inserted.
 //!
 //! If an `Occupied(entry)` is yielded, then the key *was* found. In this case,
-//! the user has several options: they can `get`, `insert`, or `remove` the
```
Comment (frewsxcv) [inline src/libstd/collections/mod.rs]: I'm generally in favor of the [serial comma](https://en.wikipedia.org/wiki/Serial_comma), but I'm not sure if Rust has an official policy around this. Regardless, in this particular instance, I don't think there's any ambiguity in either scenario so I won't waste your time changing it.

**Review by frewsxcv — APPROVED** [review]

**Comment by frewsxcv** [comment]
Looks great, thanks! ✨ 

@bors r+ rollup

File: src/libstd/collections/mod.rs (hunk 1 above)
Comment (steveklabnik) [inline src/libstd/collections/mod.rs]: Same here, generally very pro, unsure if we've made it actual policy.

Comment (GuillaumeGomez) [inline src/libstd/collections/mod.rs]: As said on IRC: I'm for more fluidity. Commas break readings and shouldn't be more present than necessary.

**Pull request merged**